packages/venia-ui/lib/components/Country/country.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { func, shape, string } from 'prop-types';
+import { useCountry } from '@magento/peregrine/lib/talons/Country/useCountry';
+
+import { mergeClasses } from '../../classify';
+import Field from '../Field';
+import Select from '../Select';
+import defaultClasses from './country.css';
+import { GET_COUNTRIES_QUERY } from './country.gql';
+
+const Country = props => {
+    const talonProps = useCountry({
+        queries: {
+            getCountriesQuery: GET_COUNTRIES_QUERY
+        }
+    });
+    const { countries, loading } = talonProps;
+    const { classes: propClasses, validate } = props;
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    return (
+        <Field id="country" label="Country" classes={{ root: classes.root }}>
```

Comment:
A suggestion here, to make this component useful in all use cases, take `field` as a prop. If not provided then use `country`.  The same for `Region` component.

Reply:
Thanks for writing this component, it is very useful.

Reply:
Made these adjustments in 5be51aa, good suggestion 👍